addons/project/views/project_portal_project_task_templates.xml

```diff
@@ -80,7 +80,7 @@
                                 <td>
                                     <t t-set="assignees" t-value="task.sudo().user_ids"/>
                                     <div t-if="assignees" class="flex-nowrap ps-3">
-                                        <img class="rounded o_portal_contact_img me-2" t-attf-src="#{image_data_uri(assignees[:1].avatar_128)}" alt="User" style="width: 20px; height: 20px;"/>
+                                        <img class="o_avatar o_portal_contact_img rounded" t-attf-src="#{image_data_uri(assignees[:1].avatar_128)}" alt="User"/>
```

Comment:
The SCSS for the o_portal_contact_img class has been removed in the commit mentioned below, so this class is no longer needed. [commit](https://github.com/odoo/odoo/commit/df8535fbd40e1e5c09dbe616a3332c76c23525c8#diff-06bd87f24b5c2bd7488abf9e9aadf350c918c5dd57994e7a8c6612cd640a7fabL367)